src/huggingface_hub/templates/modelcard_template.md

```diff
@@ -32,6 +32,12 @@
 - **Paper [optional]:** {{ paper | default("[More Information Needed]", true)}}
 - **Demo [optional]:** {{ demo | default("[More Information Needed]", true)}}
 
+## How to Get Started with the Model
+
+Use the code below to get started with the model.
+
```

Comment:
The code within the *How to get started*  can get quite long so keeping the ***Click to expand*** would be help to condense it, especially with the proposed moving to this high in the MC